proposals/lip-0044.md

```diff
@@ -570,13 +540,10 @@ def getGeneratorsBetweenTimestamps(startTimestamp: uint32, endTimestamp: uint32)
     if endTimestamp < startTimestamp:
         raise Exception('End timestamp cannot be smaller than start timestamp.')
 
-    if startTimestamp < genesisDataStore.timestamp:
-        raise Exception('Start timestamp cannot be smaller than genesis block timestamp.')
-
     result = {}
 
-    startSlotNumber = floor((startTimestamp - genesisDataStore.timestamp) / BLOCK_TIME) + 1
-    endSlotNumber = floor((endTimestamp - genesisDataStore.timestamp) / BLOCK_TIME) - 1
+    startSlotNumber = floor(startTimestamp / BLOCK_TIME) + 1
```

Comment:
We should avoid using `floor` and instead use integer division (`//`).

Reply:
changed everywhere in LIP 0044 and LIP 0058.